packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/property-type-form/expected-value-selector.tsx

```diff
@@ -163,7 +175,8 @@ const ExpectedValueSelector: ForwardRefRenderFunction<
             return (
               <ExpectedValueChip
                 {...getTagProps({ index })}
-                key={typeId}
+                // eslint-disable-next-line react/no-array-index-key
+                key={index}
```

Comment:
You can probably use something like `typeof expectedValue === "object" ? expectedValue.id : expectedValue` here

Reply:
Thank you, followed your suggestion in 2ad67387d9c8f0453ecf2213867beda796492e5e